Resprites for machetes and some xenoarcheology equipment

SealCure opened the pull request:
<!-- 
Do not forget to add a changelog when you have made admin/player facing changes that can alter gameplay.
Examples which require a changelog entry include:
* Adding/removing objects that players may interact with, or the way they function.
* Adding/removing/altering admin tools.
* Changing the map.

Examples were changelog entries are optional/not typically required:
* Cosmetic changes such as descriptions, sound effects, etc.
* Optimizations and other changes to underlying systems which do not affect gameplay.
* Minor bug fixes.

You'll find a README and example file in .\html\changelogs\ for further instructions.

You can also find a template for adding your changelog directly to the PR description here: https://github.com/Baystation12/Baystation12/wiki/Automatic-changelog-generation
-->
Readding some of the sprites from the previous bulk change, now in nice, manageable chunks instead of one big mess.
This part resprites machetes, along with some xenoarcheological equipment. Resprited items include:
- All types of machete, including the Pathfinder's Deluxe machete
- Stasis cage

An overview of all sprites can be seen below:

![](https://i.gyazo.com/c9f8361c58a8509325735972d40aab4a.png)

As before, all sprites have been made by Pebbles.

:cl:
imageadd: Resprites machetes and stasis cages with sprites by Pebbles.
/:cl:

SierraKomodo reviewed (changes requested):
- `stasis_cage_on` has a random flickering white pixel in the top left
- `excavation` seems to just be the same thing, but slightly smaller. Smaller sprites are a bit more awkward to see/click in practice, not sure if this provides any real improvement.

SealCure commented:
Old pickset holder is back, as for the white pixel on the stasis cage, I think this might be an issue with the diffs. There's no white pixel ingame, and it renders a white pixel in the diff for the old stasis cage as well.

comma commented:
Anomaly container is overlaid over the anomaly, if the sprite has gap at the bottom it might lead to awkward peeking out for some anomaly sprites e.g. the dog penis one

SierraKomodo reviewed (changes requested):
Per comma's comment, I just checked and it does look a tad odd with the larger anomalies that use the full tile for their sprite. Either the anomaly container will have to be adjusted to accommodate the full-size sprites, or the existing anomaly sprites will also need to be adjusted.

SealCure committed:
Stasis cage, anomaly container and excav pick holder sprites


Sprites for all machetes


remove flickering pixel?


Return old sprite


Restore old anomaly container

SealCure commented:
I think adjusting the sprite myself would probably go badly, and updating all the anomaly sprites is probably a matter for a future PR, so for now I'll just nix the new container sprite.
